nexus/src/external_api/device_auth.rs

```diff
@@ -125,7 +125,6 @@ pub struct DeviceAuthVerify {
 }]
 pub async fn device_auth_verify(
     rqctx: Arc<RequestContext<Arc<ServerContext>>>,
-    _params: Query<DeviceAuthVerify>,
```

Comment:
We're not doing anything with these on the server, and we're going to change it to manual entry anyway: https://github.com/oxidecomputer/console/issues/999